website/docs/r/eks_cluster.html.markdown

```diff
@@ -81,6 +81,8 @@ resource "aws_iam_role_policy_attachment" "example-AmazonEKSVPCResourceControlle
 
 -> The below configuration uses [`depends_on`](/docs/configuration/resources.html#depends_on-explicit-resource-dependencies) to prevent ordering issues with EKS automatically creating the log group first and a variable for naming consistency. Other ordering and naming methodologies may be more appropriate for your environment.
 
+-> Be carreful the logroup name's /aws/eks/${var.cluster_name}/cluster is mandatory until EKS API accept loggroup name's as parameter.
```

Comment:
I would remove this as a note, but add a line as a comment above the attribute name, eg:
```
resource "aws_cloudwatch_log_group" "example" {  
  # The log group name format is /aws/eks/<cluster-name>/cluster
  # Reference: https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html
  name              = "/aws/eks/${var.cluster_name}/cluster"
```